### PR TITLE
Fixes for TintScreen and tint pictures

### PR DESCRIPTION
**Pull request by fmatthew5876**

### CODE REVIEW

**Comment by fmatthew5876** [comment]
Fixed #1546 by re-ordering `Update()` calls.

File: src/tone.h [hunk 1]
```diff
@@ -71,4 +61,23 @@ class Tone {
 	int gray;
 };
 
+inline Tone blend(const Tone& l, const Tone& r) {
```
Comment (Ghabry) [inline src/tone.h]: Could I have this instead either

a) As a static/factory function in the Tone or
b) As a method on the Tone object?

**Comment by fmatthew5876** [comment]
Renamed `blend()` to `Blend()`, to match our coding style.

File: src/tone.h (hunk 1 above)
Comment (Ghabry) [reply src/tone.h]: well everybody always tells you about encapsulation and namespacing, so a free function looks unfamiliar but when you see this as some kind of operator this argumentation makes sense.

Comment (fmatthew5876) [reply src/tone.h]: The missing encapsulation is that we don't have a namespace around liblcf and Player.

**Pull request merged**